pandas/tests/frame/test_apply.py

```diff
@@ -721,33 +708,27 @@ def test_consistent_coerce_for_shapes(self):
         expected = Series([[1, 2] for t in df.itertuples()])
         assert_series_equal(result, expected)
 
-    def test_consistent_names(self):
+    def test_consistent_names(self, int_frame_const_col):
         # if a Series is returned, we should use the resulting index names
-        df = DataFrame(
-            np.tile(np.arange(3, dtype='int64'), 6).reshape(6, -1) + 1,
-            columns=['A', 'B', 'C'])
+        df = int_frame_const_col
 
         result = df.apply(lambda x: Series([1, 2, 3],
                                            index=['test', 'other', 'cols']),
                           axis=1)
-        expected = DataFrame(
-            np.tile(np.arange(3, dtype='int64'), 6).reshape(6, -1) + 1,
-            columns=['test', 'other', 'cols'])
+        expected = DataFrame(int_frame_const_col.values,
```

Comment:
Instead of using the DataFrame constructor can you just rename the columns to get to the same place?